app/code/Magento/ReviewGraphQl/Mapper/ReviewDataMapper.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\ReviewGraphQl\Mapper;
+
+use Magento\Catalog\Model\Product;
+use Magento\Review\Model\Review;
+
+/**
+ * Converts the review data from review object to an associative array
+ */
+class ReviewDataMapper
+{
+    /**
+     * Mapping the review data
+     *
+     * @param Review $review
+     *
+     * @return array
+     */
+    public function map(Review $review): array
+    {
+        return [
+            'summary' => $review->getData('title'),
+            'text' => $review->getData('detail'),
+            'nickname' => $review->getData('nickname'),
+            'created_at' => $review->getData('created_at'),
+            'rating_votes' => $review->getData('rating_votes'),
```

Comment:
Great work @eduard13. very nice PR! I just have a couple of comments.
is rating_votes being used anywhere?

Reply:
Yeah, looks like after refactoring the code, I've missed to remove this one.
Thank you for pointing up.
Will wait for the feedback regarding the other points and will make all the needed adjustments at once.

Reply:
Removed this one.